src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatiable" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scaled=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+        Hello World!
```

Comment:
I think indentation here is a bit too much.

```suggestion
    Hello World!
```